### PR TITLE
Release2.0.6 drill 4

### PR DESCRIPTION
**Pull request by patsevanton**

### CODE REVIEW

File: spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java [hunk 1]
```diff
@@ -68,30 +68,41 @@
     private Set<Specialty> specialties;
 
     public Integer getId() {
```
Comment (Oksana2020Gavrilova) [inline spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java]: need to delete changes

File: spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java [hunk 2]
```diff
@@ -68,30 +68,41 @@
...
     public String getFirstName() {
         return this.firstName;
     }
 
+    private void printId(Integer id){
+        System.out.println("0.4.0");
+        System.out.println(id);
+    }
+
     public void setFirstName(String firstName) {
```
Comment (Oksana2020Gavrilova) [inline spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java:87]: need to delete changes

File: spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java [hunk 3]
```diff
@@ -68,30 +68,41 @@
...
+        System.out.println("0.4.0");
         this.firstName = firstName;
     }
 
     public String getLastName() {
```
Comment (Oksana2020Gavrilova) [inline spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java]: need to delete changes

File: spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java [hunk 4]
```diff
@@ -68,30 +68,41 @@
...
+        System.out.println("0.4.0");
         return this.lastName;
     }
 
     public void setLastName(String lastName) {
```
Comment (Oksana2020Gavrilova) [inline spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java]: need to delete changes

File: spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java [hunk 5]
```diff
@@ -68,30 +68,41 @@
...
+        System.out.println("0.4.0");
         this.lastName = lastName;
     }
 
     protected Set<Specialty> getSpecialtiesInternal() {
```
Comment (Oksana2020Gavrilova) [inline spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java]: need to delete changes

File: spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java [hunk 6]
```diff
@@ -100,16 +111,19 @@ public void setLastName(String lastName) {
 
     @XmlElement
     public List<Specialty> getSpecialties() {
```
Comment (Oksana2020Gavrilova) [inline spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java]: need to delete changes

File: spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java [hunk 7]
```diff
@@ -100,16 +111,19 @@ public void setLastName(String lastName) {
...
+        System.out.println("0.4.0");
         List<Specialty> sortedSpecs = new ArrayList<>(getSpecialtiesInternal());
         PropertyComparator.sort(sortedSpecs, new MutableSortDefinition("name", true, true));
         return Collections.unmodifiableList(sortedSpecs);
     }
 
     public int getNrOfSpecialties() {
```
Comment (Oksana2020Gavrilova) [inline spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java]: need to delete changes

File: spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java [hunk 8]
```diff
@@ -100,16 +111,19 @@ public void setLastName(String lastName) {
...
+        System.out.println("0.4.0");
         return getSpecialtiesInternal().size();
     }
 
     public void addSpecialty(Specialty specialty) {
```
Comment (Oksana2020Gavrilova) [inline spring-petclinic-vets-service/src/main/java/org/springframework/samples/petclinic/vets/model/Vet.java:117]: need to delete changes

**Review by Oksana2020Gavrilova — CHANGES_REQUESTED** [review]

**Comment by patsevanton** [comment]
Fixed https://github.com/Drill4J/spring-petclinic-microservices/commit/c4418eff6bfd0408c3f074e9cea21029cf204e4e

**Pull request closed without merging**